lib/bridge/IrcHandler.js

```diff
@@ -448,11 +448,12 @@ IrcHandler.prototype.onMode = Promise.coroutine(function*(req, server, channel,
 IrcHandler.prototype.onModeIs = Promise.coroutine(function*(req, server, channel, mode) {
     req.log.info(`onModeIs for ${channel} = ${mode}.`);
 
-    mode = mode.slice(1); // Remove '+'
-
     // Delegate to this.onMode
     let promises = mode.split('').map(
         (modeChar) => {
+            if (modeChar !== '+') {
```

Comment:
You almost certainly mean `===` here. You want to ignore `+` and fallthrough to handling everything else.